node/src/components/event_stream_server/sse_server.rs

```diff
@@ -314,10 +314,22 @@ fn create_422() -> Response {
     response
 }
 
+/// Creates a 204 response (No Content) to be returned if the server has too many subscribers.
+///
+/// HTTP 204 is mentioned in
+/// https://html.spec.whatwg.org/multipage/server-sent-events.html#server-sent-events-intro as a
+/// suitable way to tell the client to stop reconnecting.
+fn create_204() -> Response {
```

Comment:
I'm not sure 204 is the right response type here, as it is very misleading. The correct HTTP error code, in my opinion, is [HTTP 503](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/503). If you are concerned about reconnections, maybe set the [`Retry-After` header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Retry-After)?